src/dosbox.cpp

```diff
@@ -543,6 +540,9 @@ void DOSBOX_Init()
 	        "               some games may not use them properly (flickering) or may need\n"
 	        "               more system memory to use them.");
 
+	Pbool = secprop->Add_bool("vga_8dot_font", only_at_start, false);
```

Comment:
This `Pbool` variable name is confusing both due to the name and due to the Almostcamelcase style.

Can you add `Prop_bool* pbool` in the beginning of this function, just like we have `pstring` and `pint` already?

Reply:
Finally squashed 'em all.